zopkio/utils.py

```diff
@@ -45,6 +45,15 @@ def check_file_with_exception(filename):
   if not os.path.isfile(filename):
     raise ValueError(filename + " is not a file")
 
+def check_testfile_dir_structure(filename):
+  """
+  Checks if the test file has correct directory structure for importing has module
+  Makes sure there is no directory with same testfile name to cause conflict
+  """
+  dirname =  os.path.splitext(filename)[0]
+  if os.path.isdir(dirname):
```

Comment:
this only determines that there is a file foo.py in directory foo, it will not handle the case of a different folder foo with an __init__.py that is also in the sys.path, might be worth adding a check for that